README.md

```diff
@@ -9,12 +9,17 @@ sudo dnf install hugo
 Clone this repository (or your fork).
 
 
-# Web page preview
+# Web page build and preview
 
-To build and run a preview of the website on a local server:
+To build the website:
 ```
 npm install
-hugo -e development
+npm run build
```

Comment:
I think 'npm run build' is better as it does not require hugo tool to be installed. Please note that for the preview of the website 'hugo' command line tool is necessary.

Reply:
That makes sense.